source/agora/node/Ledger.d

```diff
@@ -1680,13 +1681,21 @@ version (unittest)
                     params.GenesisTimestamp +
                     (blocks.length * params.BlockInterval.total!"seconds"));
 
+            Hash cycle_seed;
+            Height cycle_seed_height;
+            getCycleSeed(key_pair, cycle_seed, cycle_seed_height);
+            assert(cycle_seed != Hash.init);
+            assert(cycle_seed_height != Height(0));
+            auto cycle = PreImageCycle(cycle_seed, cycle_seed_height,
+                this.params.ValidatorCycle);
```

Comment:
`params` not `this.params` because `this.params` is set by the call to `super`, which occurs after this.